CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing to the Hedera SDK for JavaScript
+
+Thank you for your interest in contributing the Hedera SDK for C++!
+
+We appreciate your interest in helping us and the rest of our community. We welcome bug reports, feature requests, and code contributions.
+
+**Jump To:**
+
+- [Release New Version](#release-new-version)
+
+## Release New Version
+
+### Step 1 - Populate New Tag
+
+Prepare a new git `tag` and push it to `origin`.
+
+Add a new git `tag` using the following command in terminal where `v0.3.0` is the name of the newly added tag:
+> git tag -s v0.3.0
+
+Then execute:
+> git push origin v0.3.0
+
+It will populate the newly added tag in `origin`. 
+
+### Step 2 - Draft Release in GitHub
+
+Open [Hedera C++ SDK](https://github.com/hashgraph/hedera-sdk-cpp) and search for `Create a new release` link.
+Select tag `v0.3.0` from the dropdown list. Enter the Release title to be same as the release number which is been prepared (for example: `v0.3.0`).
+
+### Step 3 - Notes
+
+It's a good decision to tap direclty on button `Generate release notes` to get an automatically generated notes which can be used as a basis.
+
+#### What's Changed
+
+In this section should be filled all of the issues which have been completed and merged into `main` branch. Use only the title of each issue and add a prefix with ➡️.
```

Comment:
```suggestion
This section should be filled with all of the issues which have been completed and merged into `main` branch. Use only the title of each issue and add a prefix with ➡️.
```

This section also should just contain the over-arching new features and bug fixes in plain English.